terracumber_config/tf_files/SUSEManager-Test-Orion-Reporting-Hub.tf

```diff
@@ -0,0 +1,211 @@
+// Mandatory variables for terracumber
+variable "URL_PREFIX" {
+  type = "string"
+  default = "https://ci.suse.de/view/Manager/view/Manager-Test/job/manager-TEST-Orion-acceptance-tests-reportdb-hub"
+}
+
+variable "CUCUMBER_COMMAND" {
+  type = "string"
+  default = "export PRODUCT='Uyuni' && run-testsuite #TODO run only sanity, core and hub tests, including reporting"
+}
+
+variable "CUCUMBER_GITREPO" {
+  type = "string"
+  default = "https://github.com/uyuni-project/uyuni.git"
+}
+
+variable "CUCUMBER_BRANCH" {
+  type = "string"
+  default = "hub-reporting-beta3"
+}
+
+variable "CUCUMBER_RESULTS" {
+  type = "string"
+  default = "/root/spacewalk/testsuite"
+}
+
+variable "MAIL_SUBJECT" {
+  type = "string"
+  default = "Results TEST-HUB $status: $tests scenarios ($failures failed, $errors errors, $skipped skipped, $passed passed)"
+}
+
+variable "MAIL_TEMPLATE" {
+  type = "string"
+  default = "../mail_templates/mail-template-jenkins.txt"
+}
+
+variable "MAIL_SUBJECT_ENV_FAIL" {
+  type = "string"
+  default = "Results TEST-HUB : Environment setup failed"
+}
+
+variable "MAIL_TEMPLATE_ENV_FAIL" {
+  type = "string"
+  default = "../mail_templates/mail-template-jenkins-env-fail.txt"
+}
+
+// sumaform specific variables
+variable "SCC_USER" {
+  type = "string"
+}
+
+variable "SCC_PASSWORD" {
+  type = "string"
+}
+
+variable "GIT_USER" {
+  type = "string"
+  default = null // Not needed for master, as it is public
+}
+
+variable "GIT_PASSWORD" {
+  type = "string"
+  default = null // Not needed for master, as it is public
+}
+
+provider "libvirt" {
+  uri = "qemu+tcp://salzbreze.mgr.suse.de/system"
+}
+
+module "base_core" {
+  source = "./modules/base"
+
+  cc_username = var.SCC_USER
+  cc_password = var.SCC_PASSWORD
+
+  images = ["centos7o", "opensuse152o", "opensuse153o","sles15sp3o", "sles15sp4o"]
+  use_avahi    = false
+  name_prefix  = "suma-reportdb-"
```

Comment:
I think this should have something to do with the tf file name.

We have "suma-testorion-" prefix for the "SUSEManager-Test-Orion-NUE.tf".
Now we have testorion and reportdb.
Should we limit this testsetup to our squad, or better see it as special setup for testing hub/reporting?

What about:
SUSEManager-Test-Hub.tf with the prefix "suma-testhub-" ?

Reply:
This is for testing hub or for testing reporting tool on hub?
If we have special harware and environment, including a special pipeline for that feature, I see fine to have something like `SUSEManager-Test-Hub.tf` but then please change also the pipeline name to `manager-TEST-Hub-acceptance-tests`

Reply:
I'll go with `SUSEManager-Test-Hub.tf` , prefix: `suma-testhub-` and pipeline `manager-TEST-Hub-acceptance-tests` .